h/views/accounts.py

```diff
@@ -110,12 +110,18 @@ def get(self):
     def _js_config(self) -> dict[str, Any]:
         csrf_token = get_csrf_token(self.request)
 
+        flash_messages = []
+        for queue in ["success", "error"]:
+            for msg in self.request.session.pop_flash(queue):
+                flash_messages.append({ "type": queue, "message": msg })
+
         js_config = {
             "styles": self.request.registry["assets_env"].urls("forms_css"),
             "csrfToken": csrf_token,
             "features": {
                 "log_in_with_orcid": self.request.feature("log_in_with_orcid"),
             },
+            "flashMessages": flash_messages
```

Comment:
The content of this config overlaps with that of the signup views. We'll probably want to extract it into some shared utility or service in future.